llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -901,6 +901,48 @@ def FeaturePkFmacF16Inst : SubtargetFeature<"pk-fmac-f16-inst",
   "Has v_pk_fmac_f16 instruction"
 >;
 
+def FeatureVCUBEInsts : SubtargetFeature<"V_CUBE-Insts",
+  "HasVCUBEInsts",
+  "true",
+  "Has V_CUBE* instructions"
+>; 
+
+def FeatureVLERPInsts : SubtargetFeature<"V_LERP-insts",
+  "HasVLERPInsts",
+  "true",
+  "Has V_LERP* instructions"
+>; 
+
+def FeatureVSADInsts : SubtargetFeature<"V_SAD-insts",
+  "HasVSADInsts",
+  "true",
+  "Has V_SAD* instructions"
+>; 
+
+def FeatureVQSADInsts : SubtargetFeature<"V_QSAD-insts",
+  "HasVQSADInsts",
+  "true",
+  "Has V_QSAD* instructions"
+>; 
+
+def FeatureVCVTNORMInsts : SubtargetFeature<"V_CVT_NORM-insts",
+  "HasVCVTNORMInsts",
+  "true",
+  "Has V_CVT_NORM* instructions"
+>;
+
+def FeatureVCVTPKNORMVOP2Insts : SubtargetFeature<"V_CVT_PKNORM-VOP2-insts",
+  "HasVCVTPKNORMVOP2Insts",
+  "true",
+  "Has V_CVT_NORM* VOP2 instructions"
```

Comment:
This is a weird name/description because `v_cvt_pk_norm_i16_f32`/`v_cvt_pk_norm_u16_f32` are only VOP2 on GFX6-7, they are VOP3 only on newer hardware. 

Maybe something like `Has V_CVT_PK_NORM_*_F32 instructions`/`Has V_CVT_PK_NORM_*_F16 instructions` would be better.